test/v200/utils/common/parent_command_test_utils.go

```diff
@@ -0,0 +1,205 @@
+package common
+
+import (
+	"fmt"
+
+	schema "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
+)
+
+// parentCommandAdded adds a new command to the test schema data
+func (testDevFile *TestDevfile) parentCommandAdded(command schema.CommandParentOverride) {
+	LogInfoMessage(fmt.Sprintf("parent command added Id: %s", command.Id))
+	testDevFile.SchemaDevFile.Parent.Commands = append(testDevFile.SchemaDevFile.Parent.Commands, command)
+}
+
+// addParentEnv creates and returns a specifed number of env attributes in a schema structure
+func addParentEnv(numEnv int) []schema.EnvVarParentOverride {
+	commandEnvs := make([]schema.EnvVarParentOverride, numEnv)
+	for i := 0; i < numEnv; i++ {
+		commandEnvs[i].Name = "Name_" + GetRandomString(5, false)
+		commandEnvs[i].Value = "Value_" + GetRandomString(5, false)
+		LogInfoMessage(fmt.Sprintf("Add Parent Env: %s", commandEnvs[i]))
+	}
+	return commandEnvs
+}
+
+// addParentGroup creates and returns a group in a schema structure
+func (testDevFile *TestDevfile) addParentGroup() *schema.CommandGroupParentOverride {
+
+	commandGroup := schema.CommandGroupParentOverride{}
+	commandGroup.Kind = schema.CommandGroupKindParentOverride(GetRandomValue(GroupKinds).String())
+	kind := schema.CommandGroupKind(commandGroup.Kind)
+	LogInfoMessage(fmt.Sprintf("parent group Kind: %s, default already set %t", commandGroup.Kind, testDevFile.GroupDefaults[kind]))
+	// Ensure only one and at least one of each type are labelled as default
+	if !testDevFile.GroupDefaults[schema.CommandGroupKind(kind)] {
+		testDevFile.GroupDefaults[schema.CommandGroupKind(kind)] = true
+		commandGroup.IsDefault = true
+	} else {
+		commandGroup.IsDefault = false
+	}
+	LogInfoMessage(fmt.Sprintf("parent group isDefault: %t", commandGroup.IsDefault))
+	return &commandGroup
+}
+
+// AddParentCommand creates a command of a specified type in a schema structure and pupulates it with random attributes
```

Comment:
I'll update this.  I agree attributes is an overloaded term and it's used in other comments as well.  I'll try to catch those in future changes and replace it with `property` since that's the term the schema uses.  In this case, I replaced `attributes` with `values` since that's what's being randomly generated